package.json

```diff
@@ -11,7 +11,7 @@
     ],
     "dependencies": {
         "got-cjs": "12.0.0-beta.3",
-        "header-generator": "^1.0.0",
+        "header-generator": "^1.1.0-beta.1",
```

Comment:
we should be careful with depending on non stable things, we already got burned recently with a beta of our own package (browser-pool), many actors are not using lock files and would use the latest beta automatically on next run

i am quite fine with depending on beta, but i'd suggest to pin it

cc @mnmkng

Reply:
Yeah, we're aware. We did not plan to release it with `beta`. But it's a good point that pinning it is much safer anyway.